core/src/main/java/org/stellar/anchor/util/ErrorHelper.java

```diff
@@ -0,0 +1,18 @@
+package org.stellar.anchor.util;
+
+import static org.stellar.anchor.util.Log.*;
+
+public class ErrorHelper {
```

Comment:
Generally, I would avoid doing log and throw. We should let the code catching the exception decide if they want to log the error.

Reply:
I remember a discussion that we would like to log error and throw and this logic is prevalent in the code. I am ok with either logging at the catcher or both. But we should not go back and forth.

The logic was in the original code. If we decide to avoid log and throw, I can create a ticket for that purpose.

Ex: [RestRateIntegration.java](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/platform/src/main/java/org/stellar/anchor/platform/callback/RestRateIntegration.java#L71)
[JwtService#L235](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/core/src/main/java/org/stellar/anchor/auth/JwtService.java#L234), [Sep1Service#62](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/core/src/main/java/org/stellar/anchor/sep1/Sep1Service.java#L62)

And some other places.

Reply:
https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/platform/src/main/java/org/stellar/anchor/platform/callback/RestRateIntegration.java#L71

`sout` removed. 

I am not a fan of using reflection either because it impacts the performance. However, in this case, performance is not a concern because we are throwing exception. But if we decide not to use log and throw pattern, this function can be removed. 

We may take care of the log and throw pattern in a different ticket.

In this particular use case, only `ServerErrorException` is thrown. We remove the function and just throw the exception

Reply:
Ticket created for future improments.

[ANCHOR-779](https://stellarorg.atlassian.net/browse/ANCHOR-779)

Reply:
If we still do logging + throw in this PR, can we at least inline this method and not introduce a reflection helper in the codebase?